nodejs/week1/homework/src/backend/app.js

```diff
@@ -0,0 +1,54 @@
+const express = require("express");
+const app = express();
+
+// import data here
+const meals = require("./data/meals.json");
+const reviews = require("./data/reviews.json");
+const reservations = require("./data/reservations.json");
+
+//responds with all meals (with their reviews, if any) from data file
+const mealsWithReviews = meals.map(meal => {
+  const mealReview = reviews.filter(review => review.mealId === meal.id);
+  return Object.assign(meal, ({ "reviews": mealReview }));
```

Comment:
Why did you use Object.assign?
How it differs from meal["reviews"]=mealReview?

Reply:
Because each meal in meals json is an object. And I wanted to add a new pair of key-value to it that's why I decided to use Object.assign. Is it wrong?